src/Redux/api.tsx

```diff
@@ -776,29 +780,43 @@ const routes = {
   //inventory
   getItems: {
     path: "/api/v1/items/",
+    method: "GET",
+    TRes: Type<PaginatedResponse<InventoryItemsModel>>(),
   },
   createInventory: {
     path: "/api/v1/facility/{facilityId}/inventory/",
     method: "POST",
+    TRes: Type<InventoryLogResponse>(),
   },
   getInventoryLog: {
-    path: "/api/v1/facility",
+    path: "/api/v1/facility/{facilityId}/inventory/",
+    method: "GET",
+    TRes: Type<PaginatedResponse<InventoryLogResponse>>(),
   },
   setMinQuantity: {
     path: "/api/v1/facility/{facilityId}/min_quantity/",
     method: "POST",
+    TRes: Type<MinimumQuantityItemResponse>(),
   },
   getMinQuantity: {
-    path: "/api/v1/facility",
+    path: "/api/v1/facility/{facilityId}/min_quantity/",
     method: "GET",
+    TRes: Type<PaginatedResponse<InventorySummaryResponse>>(),
+  },
+  getMinQuantityItem: {
+    path: "/api/v1/facility/{facilityId}/min_quantity/{inventoryId}",
+    method: "GET",
+    TRes: Type<MinimumQuantityItemResponse>(),
   },
   updateMinQuantity: {
     path: "/api/v1/facility/{facilityId}/min_quantity/{inventoryId}",
```

Comment:
Missing trailing slash
```suggestion
    path: "/api/v1/facility/{facilityId}/min_quantity/{inventoryId}/",
```